src/features/landing/components/AboutUs/DeviceImage/DeviceImage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function DeviceImage() {
+  return <div>DeviceImage</div>;
```

Comment:
why you use div? don't use div, you already have a lot of MUI components, that give you possibility to create layout

Reply:
Changed